Corona-Warn-App/src/main/res/layout/include_dispatcher_card.xml

```diff
@@ -22,13 +22,25 @@
             name="icon"
             type="android.graphics.drawable.Drawable" />
 
+        <variable
```

Comment:
Would be nice to convert this into a custom view class so the preview actually shows the cards layout

Reply:
Yes, I thought about it too, but I would propose we do it after the last sprint when we start refactoring and bug fixing anyway.